README.md

```diff
@@ -32,8 +32,24 @@ $ bundle install
 
 ## Setup Authentication
 
-Need to set environment variables for login with github.
-Obtain the app_id and secret_token from https://github.com/settings/applications
+You'll need to set environment variables for login with GitHub. To do so,
+visit https://github.com/settings/applications.
+
+Set up your app with these values:
```

Comment:
But if you've cloned the repo, you're not really _creating_ the app; 'rails new' has already been run. You're adding to the already created app.